### PR TITLE
Merge dev into Master

### PR DESCRIPTION
**Pull request by MARMOTatZJU**
apply various changes 
* large search region size (e.g. lasot/got-10k)
* new features in sot_video.py demo
  * give box coordinate
  * set scale ratio

### CODE REVIEW

File: docs/TUTORIALS/SOT_MODEL_ZOO.md [hunk 1]
```diff
@@ -15,14 +15,15 @@ VOT test configuration directory: _experiments/siamfcpp/test/vot_
 
 
 | Backbone | Pipeline | Dataset | A | R | EAO | FPS@GTX2080Ti | FPS@GTX1080Ti | Config. File |
-|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
```
Comment (lzx1413) [inline docs/TUTORIALS/SOT_MODEL_ZOO.md:18]: there should be 9 |---|, not 10?

File: docs/TUTORIALS/SOT_MODEL_ZOO.md [hunk 2]
```diff
@@ -15,14 +15,15 @@ VOT test configuration directory: _experiments/siamfcpp/test/vot_
...
 | AlexNet | SiamFCppTracker | VOT2018 |0.576 | 0.183 | 0.393| ~200| ~185 | [siamfcpp_alexnet.yaml](../../experiments/siamfcpp/test/vot/siamfcpp_alexnet.yaml)
+ | 
 | GoogLeNet | SiamFCppTracker | VOT2018 | 0.583 | 0.173 | 0.426 | ~80 | ~65 | [siamfcpp_googlenet.yaml](../../experiments/siamfcpp/test/vot/siamfcpp_googlenet.yaml) |
 
 #### Multi-template
 
 | Backbone | Pipeline | Dataset | A | R | EAO | FPS@GTX2080Ti | FPS@GTX1080Ti | Config. File |
-|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
```
Comment (lzx1413) [inline docs/TUTORIALS/SOT_MODEL_ZOO.md:26]: same with above

File: docs/TUTORIALS/SOT_MODEL_ZOO.md [hunk 3]
```diff
@@ -84,10 +85,15 @@ Augmenting the search region may further improve the performance on some benchma
 #### Large _x_size_ on LaSOT
 
 | Backbone | Pipeline | Dataset | x_size | score_size | Success | Precision | Normalized Precision | Config. File |
-|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
```
Comment (lzx1413) [inline docs/TUTORIALS/SOT_MODEL_ZOO.md:88]: same question

File: docs/TUTORIALS/SOT_MODEL_ZOO.md (hunk 1 above)
Comment (MARMOTatZJU) [reply docs/TUTORIALS/SOT_MODEL_ZOO.md]: Fixed.

File: docs/TUTORIALS/SOT_MODEL_ZOO.md (hunk 3 above)
Comment (MARMOTatZJU) [reply docs/TUTORIALS/SOT_MODEL_ZOO.md]: Fixed.

**Pull request merged**